### ISSUE TITLE
Fails to load GLTF, Validation failed, a node of the scenegraph is NULL

### ISSUE THREAD

**Issue by aWilson41** [assimp/assimp#3639]
When loading the provided file here, a hand with no animations. We get a "validation failed, a node of the scenegraph is NULL". All my other software's that support GLTF read it. Unsure why assimp can't read this one. I do have other GLTF files loading with assimp but not this one. All my files are exported from blender.

[bobsLeftHand.zip](https://github.com/assimp/assimp/files/5921688/bobsLeftHand.zip)

Anyone know any possible workarounds? Import into another software? Export? Some manual fix in the json I could do.

I'm on assimp_5.0_release branch.

**Comment by aWilson41** [comment]
Update: This only happens on my embedded GLTF files.

**Comment by contriteobserver** [comment]
having trouble reproducing this issue, attempted with tip, tags/v5.0.0 and tags/v5.0.1:
```
$ assimp info bobsLeftHand.gltf 
Launching asset import ...           OK
Validating postprocessing flags ...  OK
Importing file ...                   OK 
   import took approx. 0.44058 seconds

Memory consumption: 5926148 B
Nodes:              23
Maximum depth       6
Meshes:             1
Animations:         1
Textures (embed.):  1
Materials:          1
Cameras:            0
Lights:             0
Vertices:           6472
Faces:              3236
Bones:              21
Animation Channels: 21
Primitive Types:    triangles
Average faces/mesh  3236
Average verts/mesh  6472
Minimum point      (-0.842499 -1.505915 -0.157384)
Maximum point      (0.477401 0.143985 1.792516)
Center point       (-0.182549 -0.680965 0.817566)

Meshes:  (name) [vertices / bones / faces | primitive_types]
    0 (bob.001): [6472 / 21 / 3236 | triangle]

Named Materials:
    'middleage_cauasian_male'

Texture Refs:
    '*0'
    '*0'

Named Animations:
     'bob.001'

Node hierarchy:
bobRightHandArmature
├╴bobRightHand (mesh 0)
└╴hand_r
  ├╴__index_01_r
  │ └╴index_01_r
  │   └╴index_02_r
  │     └╴index_03_r
  ├╴__middle_01_r
  │ └╴middle_01_r
  │   └╴middle_02_r
  │     └╴middle_03_r
  ├╴__pinky_01_r
  │ └╴pinky_01_r
  │   └╴pinky_02_r
  │     └╴pinky_03_r
  ├╴__ring_01_r
  │ └╴ring_01_r
  │   └╴ring_02_r
  │     └╴ring_03_r
  └╴__thumb_01_r
    └╴thumb_01_r
      └╴thumb_02_r
        └╴thumb_03_r
```

**Comment by ts-stevem** [comment]
Loaded right up for me
<img src="https://user-images.githubusercontent.com/72423314/106826637-a6098200-66ca-11eb-94b6-55bf90fafb69.jpg" width="320" />

**Comment by RichardTea** [comment]
> I'm on assimp_5.0_release branch.

The 5.0 release branch is a year out of date (Jan 2020) and won't be getting further fixes as assimp prepares for the 5.1 release.

Please move to the "master" branch, this issue appears to have been fixed.

@kimkulling Please close this as it is fixed in current master.

**Comment by kimkulling** [comment]
Ok, thanks fot the hint @RichardTea  . Will be closed right now.
